corehq/apps/app_manager/tests/test_media_suite.py

```diff
@@ -152,7 +152,7 @@ def setUp(self):
         self.app = Application.new_app('domain', "my app")
         self.module = self.app.add_module(Module.new_module("Module 1", None))
         self.form = self.app.new_form(0, "Form 1", None)
-        self.min_spec = BuildSpec.from_string('2.21/latest')
+        self.min_spec = BuildSpec.from_string('2.21.0/latest')
```

Comment:
This one still has `latest` in it

Reply:
`from_string` parses the version and build_number(which is latest) here.